modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,80 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by organization administrators, developers, and end users; each of these roles has different privileges when connecting applications,
```

Comment:
Nit: This feels like a run on sentence. I suggest using a period :) 

```
Connected Apps can be used by organization administrators, developers, and end users**.** Each of these.....
```

> Each of these roles has different privileges when connecting applications 

What is meant by **connecting applications**? This sounds a little vague.

Reply:
```suggestion
Connected Apps can be used by organization administrators, developers, and end users.
```